aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -384,24 +400,34 @@ Tensor sparse_compressed_to_dense(
     dst_transposed.add_(to_transposed_csr);
     return dst_transposed.transpose(batch_ndim, batch_ndim + 1);
   }
-  if (self.layout() == kSparseBsr) {
-    auto crow_indices = self.crow_indices();
-    auto col_indices = self.col_indices();
+  if (self.layout() == kSparseBsr || self.layout() == kSparseBsc) {
+    Tensor compressed_indices;
+    Tensor plain_indices;
+    std::tie(compressed_indices, plain_indices) =
+        AT_DISPATCH_ROW_SPARSE_COMPRESSED_LAYOUTS(
+            self.layout(),
+            "sparse_compressed_to_dense",
+            [&]() {
+              return std::make_pair(self.crow_indices(), self.col_indices());
+            },
+            [&]() {
+              return std::make_pair(self.ccol_indices(), self.row_indices());
+            });
     auto values = self.values();
     Tensor dense = at::zeros(self.sizes(), self.options().layout(kStrided));
     if (self.dim() == 2) {
       // Pad shape so we can treat 2-d like batched, we will squeeze out the
       // phantom batch dim at the end
-      crow_indices = crow_indices.unsqueeze(0);
-      col_indices = col_indices.unsqueeze(0);
+      compressed_indices = compressed_indices.unsqueeze(0);
+      plain_indices = plain_indices.unsqueeze(0);
```

Comment:
nit: `ccol/crow/row/col_indices` method does not return a reference, right? If so, then we can `unsqueeze` in-place I presume.